packages/components/src/actions/Pagination/index.tsx

```
@@ -0,0 +1,173 @@
+            borderWidth={0}
+            key={page}
+            py="$1"
+            px="$2.5"
+            borderRadius="$2"
+            borderCurve="continuous"
+            userSelect="none"
+            pressStyle={{ bg: '$bgActive' }}
+            hoverStyle={{ bg: active ? '$bgStrong' : '$bgHover' }}
+            bg={active ? '$bgStrong' : '$transparent'}
+            onPress={() => onPageChange(page)}
+            role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)